pywinauto/remote_memory_block.py

```diff
@@ -35,18 +35,18 @@
 from __future__ import print_function
 
 import sys
-import win32api
-
 from ctypes import wintypes
 from ctypes import c_void_p
 from ctypes import pointer
 from ctypes import sizeof
 from ctypes import byref
 from ctypes import c_size_t
 from ctypes import WinError
-from . import win32functions
-from . import win32defines
-from . import win32structures
+import win32api
+
+from .windows import win32functions
```

Comment:
The whole `remote_memory_block.py` should be moved to `windows` folder. It's completely Windows only implementation for "win32" backend only.